boards/jiminy-mega256rfr2/Makefile.dep

```diff
@@ -0,0 +1,5 @@
+# export board specific dependencies to the global includes-listing
+
+#ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
+#  USEMODULE += at86rfr2
+#endif
```

Comment:
As long as this module is not available, please remove this unused dependency. What are your plans here actually? Will you adapt the existing Atmel radio driver or is a new implementation required?

Reply:
We have tested several version, i found a way to integrate it seemlessly into the AT86RF2xx driver but this requires some further testing as it seems, to loose the globel ip after some time.

Reply:
Are you sure that loosing the global IP address is caused by the radio driver? However, let's have this discussion once the PR is there.